packages/next/src/server/image-optimizer.ts

```diff
@@ -639,54 +732,52 @@ export async function imageOptimizer(
 ): Promise<{ buffer: Buffer; contentType: string; maxAge: number }> {
   const { href, quality, width, mimeType } = paramsResult
   const upstreamBuffer = imageUpstream.buffer
-  const maxAge = getMaxAge(imageUpstream.cacheControl)
-  const upstreamType =
-    detectContentType(upstreamBuffer) ||
-    imageUpstream.contentType?.toLowerCase().trim()
-
-  if (upstreamType) {
-    if (
-      upstreamType.startsWith('image/svg') &&
-      !nextConfig.images.dangerouslyAllowSVG
-    ) {
-      Log.error(
-        `The requested resource "${href}" has type "${upstreamType}" but dangerouslyAllowSVG is disabled`
-      )
-      throw new ImageError(
-        400,
-        '"url" parameter is valid but image type is not allowed'
-      )
-    }
+  const maxAge = Math.max(
+    nextConfig.images.minimumCacheTTL,
+    getMaxAge(imageUpstream.cacheControl)
+  )
```

Comment:
@ztanner Since this part is being backported (looks like its from https://github.com/vercel/next.js/pull/73909), we can also change line 818 to simply `maxAge`

Reply:
I ended up reverting this part to keep changes minimal